app/templates/navbar.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+{% block body_block %}
+    <div class="navigation">
+    <div class="nav-left">
+        Foodies
+    </div>
+    <div class="nav-right">
```

Comment:
Shouldn't it be part of base.html? Why are we creating a separate view for the nav bar?